c/src/eval_ctx.c

```diff
@@ -63,7 +82,12 @@ plcs_errors plcs_eval_ctx_set_str_eval_param(plcs_string_evaluators ix, const ch
     return PLCS_EIX_OVERFLOW;
   }
 
-  ctx.string_evaluators[ix].value = value;
+  char *maybe_dup = plcs_strdup(value);
+  if (maybe_dup == NULL) {
+    return PLCS_EALLOCATION;
+  }
+
+  ctx.string_evaluators[ix].value = maybe_dup;
```

Comment:
Is it free somewhere?

Reply:
You fixed my point in https://github.com/DataDog/dd-policy-engine/pull/29/commits/f8fb5a52092fcae1c1f26fd3c8535e29041d653c#diff-34851aba826298d51f13cb1ffb8baf8971140b0a03274fa3247d9bba2f3e666dR255-R257